docs/frost-text.md

```diff
@@ -23,6 +23,7 @@
 | `form` | `string` | `<form-owner>` | form element that this is associated with (its form owner) |
 | `hook` | `string` | `<unique-name>` | name used for testing with ember-hook |
 | `maxlength` | `number` | `<maxlength-value>` | maximum number of characters a user can enter |
+| `onClear`| `function`/`action`| empty function | triggers associated action when `X` is clicked and text is cleared |
```

Comment:
Similar documentation for `frost-button` is

onClick | string | <action-name> | triggers associated action when the button is clicked

Reply:
Ya I noticed that, but `string` doesn't make sense (in both cases). Button and this both take a `function` (action helper just returns a `function`) for `onClick` and `onClear`

Reply:
You cannot pass in "just a" `function` though so `closure action` is probably the better descriptor.  Once you settle on what to use please either a) update the rest of the components descriptions to match or b) open an issue identifying that this work needs to be done.